From 063e08206cff507ae6c4f216c13427cac618b7c9 Mon Sep 17 00:00:00 2001
From: Misty De Meo <mistydemeo@gmail.com>
Date: Fri, 25 May 2018 22:10:13 -0700
Subject: [PATCH] libr: use openssl LDFLAGS at link time

libr will have been compiled with openssl support if it was requested,
so it also needs to use the openssl LDFLAGS.

Fixes #10154.
---
 libr/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libr/Makefile b/libr/Makefile
index 0d0257708..15efadd77 100644
--- a/libr/Makefile
+++ b/libr/Makefile
@@ -45,6 +45,7 @@ endif
 ifeq ($(OSTYPE),ios)
 MLFLAGS+=-install_name @rpath/libr2.dylib
 endif
+LDFLAGS+=${SSL_LDFLAGS}
 # XXX version-script only works with shlib as output
 #ifeq ($(OSTYPE),linux)
 #PLFLAGS+=--version-script ./ld.script
-- 
2.17.0

